tests/Unit/Helpers/Evolution/DiscontinuousGalerkin/BoundaryCorrections.hpp

```diff
@@ -213,6 +201,10 @@ void test_boundary_correction_conservation_impl(
                          curved_background>::template f<System>>>,
       face_tags>;
 
+  MAKE_GENERATOR(gen);
```

Comment:
Not a change request since it's beyond the scope of this PR, but what I've learned over time is that the generators are ideally always passed in. The reason being that this makes specifying a seed much easier: you look in the `SPECTRE_TEST_CASE` for the `MAKE_GENERATOR` call and specify the seed. I think changing the `_impl` to receive the `gen` should be not too tedious, while changing the interface to take the generators, would be more annoying.

Reply:
I almost did this as part of this PR before submitting it... since you feel this way too I've added a commit. :)

Reply:
Haha noice :D Well thank you!